.travis.yml

```diff
@@ -11,7 +11,7 @@ matrix:
     - rm -rf $HOME/.m2/repository/com/sap/research/security/vulas/
     install:
     - echo 'Skipping install stage, dependencies will be downloaded during build and test stages.'
-    script: mvn -e -B -P gradle -Dspring.standalone -Dit.test='!IT01_PatchAnalyzerIT, IT*, *IT, *ITCase' -DfailIfNoTests=false --settings .travis/settings.xml clean install
+    script: mvn -e -B -P gradle -Dspring.standalone -Dit.test='!IT01_PatchAnalyzerIT, IT*, *IT, *ITCase' -DfailIfNoTests=false --settings .travis/settings.xml clean install com.github.spotbugs:spotbugs-maven-plugin:3.1.11:spotbugs
```

Comment:
Please use the same filter as in the root pom.xml (thereby exluding generated Java code). The advantage of invoking it like you did is that Spotbugs is also executed for those modules not inheriting from this parent POM (thus, rest-xxx).

Reply:
In other words: Please call `-Dspotbugs.excludeFilterFile=findbugs-exclude.xml -Dspotbugs.failOnError=false com.github.spotbugs:spotbugs-maven-plugin:3.1.11:check`, we can discuss the value of `failOnError` depending on the current state of the checks.

Reply:
I added the exclude filter in the travis job as well as the `failOnError=false`. As to when the check should fail, I think the goal should still fail when high priority bugs (as classified by spotbugs) are found (however, this is not a feature that's implemented in the spotbugs maven plugin yet).